### ISSUE TITLE
Docusaurus Issues

### ISSUE THREAD

**Issue by ichim-david** [eea/volto-eea-design-system#47]
- [x] Showcase iframe needs a min height boost, otherwise the body content is clipped
  https://eea.github.io/volto-eea-design-system/docs/Templates/PageHeader/
-  [x] Foreground color is invisible in grays swatches
   https://eea.github.io/volto-eea-design-system/docs/Guidelines/colours/
- [ - ] On the dark theme the green link color should be tweaked for better color contrast
   EDIT: dark theme toggle removed instead of changing the colors
  https://eea.github.io/volto-eea-design-system/docs/whatsnew/
- [x] Routing issues:
  https://eea.github.io/volto-eea-design-system/docs/Components/Tab/
  https://eea.github.io/volto-eea-design-system/docs/Components/Inpage-Navigation/
  https://eea.github.io/volto-eea-design-system/docs/Components/Item/
### Feb 22 feedback - tasks
- [ ] For each component add **Test** tab link that takes you to the Storybook of that component

**Comment by ichim-david** [comment]
- Showcase iframe needs a min height boost, otherwise the body content is clipped
  https://eea.github.io/volto-eea-design-system/docs/Templates/PageHeader/
-  Foreground color is invisible in grays swatches
   https://eea.github.io/volto-eea-design-system/docs/Guidelines/colours/
- On the dark theme the green link color should be tweaked for better color contrast
  https://eea.github.io/volto-eea-design-system/docs/whatsnew/
- Routing issues:
  https://eea.github.io/volto-eea-design-system/docs/Components/Tab/
  https://eea.github.io/volto-eea-design-system/docs/Components/Inpage-Navigation/
  https://eea.github.io/volto-eea-design-system/docs/Components/Item/
